### PR TITLE
Nn 1180 offender cell history api

### PR DESCRIPTION
**Pull request by Nimphal**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import java.time.LocalDateTime;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Data
+public class BedAssignment {
```
Comment (matt-crowson-moj) [inline src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java:21]: Are you going to add some Swagger docs/annotations?

File: src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    private Long bookingId;
+    private Long livingUnitId;
+    private LocalDate assignmentDate;
```
Comment (matt-crowson-moj) [inline src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java:24]: Do you need to separate the date and times into separate fields?

**Review by matt-crowson-moj — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java (hunk 2 above)
Comment (Nimphal) [reply src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java:24]: They are returned like that from the mapping and we may end up using them like this. I'll probably update at some point as we've had a play.

File: src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java (hunk 1 above)
Comment (Nimphal) [reply src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java:21]: That's a good idea, will do in a following PR, I know we have more things in the pipeline around this.

**Pull request merged**